server/services/crmDisplayService.test.ts

```diff
@@ -303,30 +310,36 @@ describe('CRM Display Service', () => {
               {
                 label: 'Is your client under 18 years old?',
                 value: 'No',
+                apiField: 'CapitalDetails.isUnder18',
               },
               {
                 label:
                   "Does your client or partner (if living with client as couple) get Income Support, Income Based Job Seeker's Allowance, Income Related Employment and Support Allowance or Guarantee State Pension Credit?",
                 value: 'Yes',
+                apiField: 'CapitalDetails.hasIncomeSupport',
               },
               {
                 label: 'How many dependants does your client have?',
                 value: 2,
+                apiField: 'CapitalDetails.numOfDependants',
```

Comment:
Suggestion -> `numberOfDependents`

Reply:
numOfDependants is correct and that is what is returned by the API.